frontend/simple/views/Invite.vue

```diff
@@ -133,12 +131,14 @@ export default {
 
       try {
         const contractId = await namespace.lookup(this.searchUser)
+        const state = await latestContractState(contractId)
         if (!this.members.find(member => member.name === this.searchUser)) {
-          this.members.push({ name: this.searchUser, contractId })
+          this.members.push({ name: this.searchUser, picture: state.attributes.picture, contractId })
```

Comment:
Change to:

```js
this.members.push({ contractId, state })
```

Then the conditional can be:

```js
if (!this.members.find(member => member.state.attributes.name === this.searchUser)) {
```

And this way any additional attributes can be easily hooked up in the UI without modify this code.